static/lang/re-en.json

```diff
@@ -2356,6 +2356,11 @@
                     "Note": "The target is @UUID[Compendium.pf2e.conditionitems.sDPxOjQ9kx2RZE8D]{Fleeing} for 1 round."
                 }
             },
+            "Undine": {
+                "FluidContortionist": {
+                    "Note": "When you roll a critical failure on a check to Squeeze, you get a failure instead, and when you roll a success, you get a critical success instead."
+                }
```

Comment:
I'm happy to move it, but this one seems unique. The others are handled by "PF2E.SpecificRule.Goblin.ExtraSquishy.Note", which I agree could be moved but it would require migration.

Reply:
I've been trying to avoid at least making the problem worse and I could see a heritage or NPC or whatnot using this. We can always migrate later when we have the time to clean the problem up.

Reply:
We can change the key if you have a better idea.